stats/fixtures/pytest_fixtures.py

```diff
@@ -0,0 +1,146 @@
+import pytest
+from datetime import datetime, timedelta
+
+import pytz
+from decimal import Decimal
+from django.contrib.auth import get_user_model
+from rest_framework.test import APIClient
+
+from silver.models import Invoice, Transaction, Subscription, BillingLog
+from silver.tests.factories import TransactionFactory, InvoiceFactory, DocumentEntryFactory, \
+    SubscriptionFactory, PlanFactory, ProviderFactory, CustomerFactory
+
+UserModel = get_user_model()
+
+
+@pytest.fixture()
+def user():
+    return UserModel.objects.create_superuser(
+        username='test', email='test@...', password='top_secret')
+
+
+@pytest.fixture()
+def api_client(user):
+    client = APIClient()
+    client.force_authenticate(user)
+    return client
+
+
+def create_customer():
```

Comment:
This is not marked with `@pytest.fixture()`. All these fixtures return more than one object, although the name suggests a single object. Also I think you can remove `create_` from their names.
You should also try to be consistent, some of these fixtures return, some don't.